services/kvstore/src/main/java/org/hyperledger/besu/services/kvstore/LayeredKeyValueStorage.java

```diff
@@ -92,6 +92,26 @@ public Optional<byte[]> get(final SegmentIdentifier segmentId, final byte[] key)
     }
   }
 
+  @Override
+  public Optional<NearestKeyValue> getNearestTo(
+      final SegmentIdentifier segmentIdentifier, final Bytes key) throws StorageException {
+    var ourNearest = super.getNearestTo(segmentIdentifier, key);
+    var parentNearest = parent.getNearestTo(segmentIdentifier, key);
```

Comment:
I'm not a fan of var. it not easy to understand the type we have

Reply:
will be explicit.  👍